packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1743,19 +1733,11 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     public async refreshLatestSummaryAck(
         proposalHandle: string | undefined,
         ackHandle: string,
-        trackerRefSeqNum: number, // back-compat summarizerNode - remove when fully enabled
         summaryLogger: ITelemetryLogger,
         version?: IVersion,
     ) {
-        if (trackerRefSeqNum < this.summaryTracker.referenceSequenceNumber) {
```

Comment:
is this right? or should we be keeping th check, and just checking against summarizer node's ref seq

Reply:
I think it should be fine to remove this.  Quick explanation on why:

Normally `refreshLatestSummaryAck` is called on every _summaryAck_ op that comes in, so the check wouldn't really be needed since op order is guaranteed in loader layer and out of order summary should not be acked by server.

Sometimes `refreshLatestSummaryAck` is called by `generateSummary` by directly asking the server for the latest, which means it could give a _summaryAck_ that is later than we are caught up to op-wise, but that would be fine since we want the latest anyway.

There could be a tight loophole where we ask server for latest and get a later _summaryAck_, and then while catching up see an older summaryAck and mistakenly think that is the latest, because we have removed this check.  If we try to summarize in that brief period, we'd be nacked anyway since previous summary handle will not match. Then while continuing to catch up in the op stream, we'd see the ack for the latest and be back in expected state again.

Even in some worst-case scenario where the latest _summaryAck_ never made it to the op stream (server issue) **and** we hit the loophole mentioned above, we wouldn't stay stuck, because we would get nacked, then retry in "safe" mode which would again ask server for latest, bringing us back up-to-date.  This time, since we are caught up, we shouldn't have to worry about getting behind again.

So I think it should correct itself and not cause corruption in all cases.